greenhouse_management/models.py

```diff
@@ -25,3 +25,23 @@ def __str__(self):
 
 class Localization(models.Model):
     name = models.CharField(max_length=100)
+
+
+class GreenHouse(models.Model):
+    name = models.CharField(max_length=255)
+    crop_type = models.CharField(max_length=255)
```

Comment:
bedziemy dawac jakies enumy? bo moze latwiej nam to bedzie pozniej filtrowac itp

Reply:
To zamienić to na
`crop_type = models.CharField(max_length=20, choices=[(tag, tag.value) for tag in CropType])`
?

Reply:
ja bym poszedł w tym kierunku:
https://docs.djangoproject.com/en/3.0/ref/models/fields/#enumeration-types

Reply:
spoko zaraz ogarne to